nodejs/awsx/cloudtrail/trail.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import * as utils from "../utils";
+
+export class Trail extends pulumi.ComponentResource {
+    /**
+     * The managed CloudWatch Log Group.
+     */
+    public readonly logGroup: aws.cloudwatch.LogGroup | undefined;
+
+    /**
+     * The managed S3 Bucket.
```

Comment:
I assume this is the S3 bucket that the trail is logged to. Do you mind updating the comment to clarify?